### PR TITLE
Updated merge_sorted_lists function

### PR DESCRIPTION
**Pull request by SydneyALV**

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
+
+    return sorted(merged)
```
Comment (laughtivity) [inline functions/merge_sorted_lists.py]: great job, we may be able to dry it up by putting everything in the return statement. 

```python
return sorted(l1 + l2)
```

**Review by laughtivity — COMMENTED** [review]
Great tests!

File: functions/merge_sorted_lists.py (hunk 1 above)
Comment (SydneyALV) [reply functions/merge_sorted_lists.py]: Thanks, Gloria! I'll do that.

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
```
Comment (laughtivity) [inline tests/merge_sorted_lists_test.py:3]:
``` python
import random
```

**Review by laughtivity — CHANGES_REQUESTED** [review]
``` python 
import random
```

File: tests/merge_sorted_lists_test.py (hunk 2 above)
Comment (SydneyALV) [reply tests/merge_sorted_lists_test.py:3]: I don't think that's a good idea. But thank you for the suggestion.

**Pull request merged**